Scripts/Editor/AssetBundleWindow.cs

```diff
@@ -112,6 +119,107 @@ void OnGUI()
+                        GameObject prefab = ((GameObject) AssetDatabase.LoadAssetAtPath(internalFilePath, typeof(GameObject)));
+                        //Debug.LogWarning(prefab == null);
+                        //AssetDatabase.ImportAsset(AssetDatabase.GetAssetPath(prefab), ImportAssetOptions.ForceUpdate);
+                        EditorUtility.SetDirty(prefab);
+                        Texture2D prefabPreview = AssetPreview.GetAssetPreview(prefab);
+                        //Debug.LogWarning(prefabPreview == null);
+
+                        // Create a temporary RenderTexture of the same size as the texture
+                        RenderTexture tmp = RenderTexture.GetTemporary(
+                                            prefabPreview.width,
+                                            prefabPreview.height,
+                                            0,
+                                            RenderTextureFormat.Default,
+                                            RenderTextureReadWrite.Default);
+                        // Blit the pixels on texture to the RenderTexture
+                        Graphics.Blit(prefabPreview, tmp);
+                        // Backup the currently set RenderTexture
+                        RenderTexture previous = RenderTexture.active;
+                        // Set the current RenderTexture to the temporary one we created
+                        RenderTexture.active = tmp;
+                        // Create a new readable Texture2D to copy the pixels to it
+                        Texture2D myTexture2D = new Texture2D(prefabPreview.width, prefabPreview.height);
+                        // Copy the pixels from the RenderTexture to the new Texture
+                        myTexture2D.ReadPixels(new Rect(0, 0, tmp.width, tmp.height), 0, 0);
+                        myTexture2D.Apply();
+                        // Reset the active RenderTexture
+                        RenderTexture.active = previous;
+                        // Release the temporary RenderTexture
+                        RenderTexture.ReleaseTemporary(tmp);
+                        byte[] bytes = myTexture2D.EncodeToPNG();
+                        imgPreviews[nameWOext] = bytes;
+                        imgPreviews[nameWOext.ToLower()] = bytes;
+
+
+
+                        if (!Directory.GetDirectories(Application.dataPath + assetBundleFolderLocation).Contains(nameWOext))
+                        {
+                            Directory.CreateDirectory(Application.dataPath + assetBundleFolderLocation + "/" + nameWOext);
+                            //File.Create(Application.dataPath + assetBundleFolderLocation + "/" + nameWOext + ".meta");
+                        }
+                        // move file
+                        File.Copy(asset, Application.dataPath + assetBundleFolderLocation + "/" + nameWOext + "/" + "model" + ".prefab", true);
```

Comment:
This is the part that moves the prefabs into their own folders on the /AssetBundles level